rpm/pam.spec

```diff
@@ -73,6 +73,7 @@ BuildRequires: autoconf >= 2.60
 BuildRequires: automake, libtool
 BuildRequires: bison, flex, sed
 BuildRequires: perl, pkgconfig, gettext-devel
+BuildRequires: pkgconfig(libcrypt)
```

Comment:
Is there a configure option to force this need also? Having it option in build env that it builds with/without based on if availble in environment is quite nasty, thus explicit configure --enable-libcrypt or something would be better.